packages/public/server/src/module/Ui/templates/static/spenden.phtml

```diff
@@ -129,11 +135,10 @@ echo $this->doctype();
                             <section>
                                 <div class="r">
                                     <div class="c24">
-                                        <p>
-                                            <img
-                                                src="https://assets.serlo.org/59f762922f93d_c1bea0db3a614ccd33887e6f615b4b60a84e642e.jpg"
-                                                alt="test" title="">
-                                        </p>
+                                        <img
+                                            id="main-image"
+                                            src="https://assets.serlo.org/59f762922f93d_c1bea0db3a614ccd33887e6f615b4b60a84e642e.jpg"
+                                            alt="test" title="">
```

Comment:
`alt="test"`? ^^

Reply:
don't know where that is coming from^^ I'll add a more usefull alt^^